src/providers/RelayProvider.ts

```diff
@@ -33,4 +60,341 @@ export class RelayProvider implements TLProvider {
       args
     )
   }
+
+  /////////////
+  // Account //
+  /////////////
+
+  /**
+   * Returns a `Promise` with the balance of given address at optionally given
+   * block tag.
+   * @param address Ethereum address.
+   * @param blockTag Block number as number or hex string or `latest` or `pending`.
+   */
+  public async getBalance(
+    address: string,
+    blockTag?: string | number | Promise<ethers.providers.BlockTag>
+  ): Promise<ethers.utils.BigNumber> {
+    const { balance } = await this.fetchEndpoint<TxInfosRaw>(
+      `users/${address}/txinfos`
+    )
+    return ethers.utils.bigNumberify(balance)
+  }
+
+  /**
+   * Returns a `Promise` with the number of sent transactions from given
+   * address at optionally given block tag.
+   * @param address Ethereum address.
+   * @param blockTag Block number as number or hex string or `latest` or `pending`.
+   */
+  public async getTransactionCount(
+    address: string | Promise<string>,
+    blockTag?: string | number | Promise<ethers.providers.BlockTag>
+  ): Promise<number> {
+    const { nonce } = await this.fetchEndpoint<TxInfosRaw>(
+      `users/${address}/txinfos`
+    )
+    return nonce
+  }
+
+  /**
+   * Returns needed information for creating an ethereum transaction.
+   * @param address Address of user creating the transaction
+   * @returns Information for creating an ethereum transaction for the given user address.
+   *          See type `TxInfos` for more details.
+   */
+  public async getTxInfos(address: string): Promise<TxInfos> {
+    const { nonce, gasPrice, balance } = await this.fetchEndpoint<TxInfosRaw>(
+      `users/${address}/txinfos`
+    )
+    return {
+      balance: new BigNumber(balance),
+      gasPrice: new BigNumber(gasPrice),
+      nonce
+    }
+  }
+
+  ///////////////////////
+  // Blockchain Status //
+  ///////////////////////
+
+  /**
+   * Returns a `Promise` with the latest block number.
+   */
+  public async getBlockNumber(): Promise<number> {
+    return this.fetchEndpoint<number>(`blocknumber`)
+  }
+
+  /**
+   * Returns a `Promise` with the current gas price as a `BigNumber`.
+   */
+  public async getGasPrice(): Promise<ethers.utils.BigNumber> {
```

Comment:
Why did you not implement this? See line 112

Reply:
The problem is that our `/users/:userAddress/txinfos` endpoint requires the address of the user. The method `getGasPrice()` on the other hand does not allow any parameters. I guess I can use the ZERO_ADDRESS then instead? Smth like
```javascript
return this.fetchEndpoint('/users/0x0...0/txinfos')
```